src/main-container/MainContainer.module.scss

```diff
@@ -0,0 +1,5 @@
+:local {
+  .main-container {
+    outline: none;
```

Comment:
should main have 100% height?

Reply:
I thought about this but erred on the side of caution. If we add a height, I feel like we'd need to make the same call for overflow handling. But that is not always desired/required (for the Page use case, or example), which opens the door to all sorts of props to disable/enable things... 

Long story long I'd like to avoid adding more styles here, for the time being at least. Makes me wonder if we should add a MainContainer to the eventual LegacyLayout and provide the default styling we used to provide there while keeping this component more generic.